src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1885,6 +1885,9 @@ internal enum ErrorCode
         WRN_UseDefViolationOut = 8886,
         WRN_UseDefViolation = 8887,
 
+        WRN_GeneratorFailedDuringInitializationDeveloper = 8888,
```

Comment:
You'll want to move these past 8891, function pointers will be merging this eveing.